mlflow/tracking/fluent.py

```diff
@@ -379,6 +644,27 @@ def delete_run(run_id):
     Deletes a run with the given ID.
 
     :param run_id: Unique identifier for the run to delete.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Set existing run_ids to delete
+        run_ids = ["13ee9e661cbf4095a7c92cc55b4e12b4", "948fbf2d0b7f4056b3dd4914845a1e1b"]
+
+        # Delete run_ids and fetch the results.
+        # Note that runs are not actually delete, only lifecycle stage is set to "deleted"
+        for run_id in run_ids:
+            mlflow.delete_run(run_id)
+            print("run_id: {}; lifecycle_stage: {}".format(run_id,
+                mlflow.get_run(run_id).info.lifecycle_stage))
```

Comment:
I think we should provide an example that a user can execute.

```python
import mlflow

with mlflow.start_run() as run:
    mlflow.log_param("p", 0)

run_id = run.info.run_id
mlflow.delete_run(run_id)

print("run_id: {}; lifecycle_stage: {}".format(run_id, mlflow.get_run(run_id).info.lifecycle_stage))
```

Reply:
Better and simple!

Reply:
done!